libbeat/outputs/kafka/kafka.go

```diff
@@ -270,6 +272,9 @@ func (k *kafka) newKafkaConfig() (*sarama.Config, error) {
 	}
 
 	cfg.Producer.Partitioner = k.partitioner
+
+	// TODO: figure out which metrics we want to collect
+	cfg.MetricRegistry = adapter.GetGoMetrics(monitoring.Default, "libbeat.output.kafka")
```

Comment:
I would probably go with `output.kafka` and not have the libbeat part. I'm aware that we have libbeat in front for other outputs part so far. But I think outputs are so independent (and potentially not necessarly in libbeat itself (plugins) in the future), that we can remove the `libbeat` part.